### PR TITLE
foam armblade toy

### PR DESCRIPTION
**Pull request by Delimusca**
like one of those foam fingers, great for skooky maint fun like the other "toys".
credit to an0n3, or someone else in ooc idk
how it looks

### CODE REVIEW

**Comment by optimumtact** [comment]
Just dont' whinge to the admins when you get robusted

**Comment by Chocobro** [comment]
except do because that's non-antag killing and That's No Good

:+1:

**Comment by dannno** [comment]
>Just dont' whinge to the admins when you get robusted

do you think it's ok to kill people for "attacking" you with a toy sword
if you do you're wrong

**Comment by optimumtact** [comment]
I do, because if you act like an antag you get treated like one

**Comment by dannno** [comment]
are you fucking kidding me
if you hit someone with a toy sword it does no damage and doesn't even do that similar of an effect to a real one
It sounds like you're stretching the rules a bit, if you kill someone for tapping you with a toy before you can identify that you didn't get hurt at all or even appear to get hurt at all you're the shitty one

**Comment by paprka** [comment]
STOP BIKESHEDDING ABOUT THE RULES

THIS IS AWESOME SINCE THE WHOLE POINT OF TOY IS TO TROLL ANTAG HUNTERS

ADD POSTHASTE

**Comment by Steelpoint** [comment]
Can we not keep adding these toys in?

Its obnoxious and gives me a heart attack half the time someone runs up to me with a Traitor weapon and starts attacking.

**Comment by Iamgoofball** [comment]
@Steelpoint lrn2read chat logs

**Comment by TZK13** [comment]
Holy shit yes. :+1: 

Also >Thinking playing around with a toy falls under acting like an antag.

**Comment by Deantwo** [comment]
> I do, because if you act like an antag you get treated like one

This should result in an arrest instead. Even citizens arrest if it is.
But yeah, the players care not for SpaceLaw, they even kill in self defense.

A crewman should never be allowed to use deadly force unless there is no other way.
Only expectation should be if an armed group is attacking the station (Nuke Ops).

If not then why even have security and a brig?
But it is a player problem and we can't really fix that in the code. Unless we start doing an reward system

**Comment by Miauw62** [comment]
lol keep the rules discussion out of this faglerds, this is a great feature :+1:

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -372,6 +373,19 @@
...
 /*
+ * Foam armblade
+ */
+/obj/item/toy/foamblade
+	name = "Foam armblade"
+	desc = "it says \"Sternside Changs #1 fan\" on it. "
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "foamblade"
+	item_state = "arm_blade"
+	attack_verb = list("pricked", "absorbed", "gored")
```
Comment (Miauw62) [inline code/game/objects/items/toys.dm]: 100/10

Comment (Iamgoofball) [inline code/game/objects/items/toys.dm]: >not making it do "You feel a tiny prick" on attack

File: code/game/objects/items/toys.dm [hunk 2]
```diff
@@ -372,6 +373,19 @@
 			user << "<span class='warning'>It's already fabulous!</span>"
 
 /*
+ * Foam armblade
+ */
+/obj/item/toy/foamblade
+	name = "Foam armblade"
```
Comment (Cheridan) [inline code/game/objects/items/toys.dm]: name should be lowercase for consistency and proper formatting.

Comment (Delimusca) [inline code/game/objects/items/toys.dm]: oh right whoops

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (Delimusca) [inline code/game/objects/items/toys.dm]: say no to override pasta, penjab and whip this out instead :>

**Pull request merged**